src/GitHub.InlineReviews/PullRequestStatusBarPackage.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Windows;
+using System.Threading;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.ComponentModelHost;
+using GitHub.VisualStudio;
+using GitHub.InlineReviews.Services;
+using Task = System.Threading.Tasks.Task;
+
+namespace GitHub.InlineReviews
+{
+    [PackageRegistration(UseManagedResourcesOnly = true)]
+    [Guid(Guids.PullRequestStatusPackageId)]
+    [ProvideAutoLoad(Guids.GitSccProviderId)]
+    public class PullRequestStatusBarPackage : AsyncPackage
```

Comment:
Does this need to be in a separate package? For context, I've been thinking recently that we should be merging `GitHub.InlineReviews` into our main package if possible, as it's completely dependent on it. Could this go in the main package? Would there be problems with that?

Reply:
When I first wrote this functionality, I made it piggyback a ride with one of the other packages. Keeping it separate can be useful to control exactly when and how it loads.

For example, I wanted this package load when the `GitSccProvider` is available and with the option `AllowsBackgroundLoading = true`. The package it was being loaded with before was using different settings (being loaded when a solution was opened).

Reply:
The `GitHubPackage` already loads at the correct time in the correct way, btw.

Reply:
BTW, the `GitHubPackage` package uses `AllowsBackgroundLoading = true`, but we'd forgotten to use the `PackageAutoLoadFlags.BackgroundLoad` option on `[ProvideAutoLoad(...)]`. This means it uses a background thread when an executing command triggers the package load but the Main thread when the package is auto-loaded. Subtle and confusing! 😕 I've addresses this issue in #1512.